docs/api/spec.md

```diff
@@ -181,10 +184,10 @@ Since version 1.2 of the FDC3 Standard it may do so via the [`fdc3.getInfo()`](r
 ```ts
 import {compareVersionNumbers, versionIsAtLeast} from '@finos/fdc3';
```

Comment:
- Missing spaces at opening / closing braces
- Should be double quotes not single quotes

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85